make/photon/registry/Dockerfile.binary

```diff
@@ -7,4 +7,7 @@ ENV GO111MODULE auto
 WORKDIR $DISTRIBUTION_DIR
 COPY . $DISTRIBUTION_DIR
 
-RUN CGO_ENABLED=0 make PREFIX=/go clean binaries
+RUN CGO_ENABLED=0 make clean bin/registry
+
```

Comment:
why is this change needed?

Reply:
The `PREFIX` variable is not used anywhere, which leads to confusion.
Therefore, I think it is better to remove it.